config/config.go

```diff
@@ -0,0 +1,181 @@
+package config
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+	"unicode"
+)
+
+type Application struct {
+	FilePath,
+	Postcode,
+	StartTimeString,
+	EndTimeString string
+	KeywordsString []string
+	StartTime,
+	EndTime int
```

Comment:
Очень небезопасно хранить экспортированные данные в двух представлениях. Ты рискуешь нарушить консистентность. Обычно, если видишь необходимость в таком - надо прятать такие данные, заполнять из в конструкторе и использовать только внутри, 
Поэтому моя рекомендация - в конфиге хранить только в строки 
```suggestion
	StartTimeString,
	EndTimeString string
	KeywordsString []string
```

Reply:
Хороший комент. Исправил